packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -96,14 +96,13 @@ export class FastSync
     return computeStartSlotAtEpoch(this.config, this.targetCheckpoint.epoch);
   }
 
-  private getNewBlockImportTarget(fromSlot?: Slot): Slot {
-    const headSlot = fromSlot || this.chain.forkChoice.headBlockSlot();
+  private getNewBlockImportTarget(fromSlot: Slot): Slot {
```

Comment:
this is called from one single place and it had param there so this is just simply remove the optional thing